content/aips/aa1d9a276fb6-AIP-Update-Polygon-Market.md

```diff
@@ -0,0 +1,143 @@
+---
+aip: <to-be-assigned>
+title: Add Assets to Aave Polygon Market and Update WMATIC Risk Parameters
+status: Proposed
+author: Coder Dan (@coderdannn), MatthewGraham, tomuky
+shortDescription: Update Polygon Market
+discussions: https://governance.aave.com/t/arc-list-aavegotchi-ghst-token-as-collateral-on-matic-aave-network/3619
+  https://governance.aave.com/t/listing-proposal-add-v2-market-assets-to-polygon-market-that-are-relevant-to-polygon-network/5511
+  https://governance.aave.com/t/arc-polygon-market-wmatic-risk-parameter-changes/6021
+created: 2021-23-11
```

Comment:
This date seems incorrect? (yyyy-mm-dd)
Could you also update the filename accordingly (using the generate-filename script)?